nnsvs/model.py

```diff
@@ -74,14 +70,52 @@ def __init__(self, in_dim, hidden_dim, out_dim, num_layers=1, bidirectional=True
         super(LSTMRNN, self).__init__()
         self.hidden_dim = hidden_dim
         self.num_layers = num_layers
+        self.relu = nn.ReLU()
```

Comment:
Looks like this is not used anymore?

Reply:
I deleted unnecessary ReLU in the latest commit.